index.html

```diff
@@ -0,0 +1,14 @@
+  <!DOCTYPE html>
+  <html lang="en">
+
+  <head>
```

Comment:
Required Changes:
- [ ] For coding on HTML you don't have to leave that many empty lines from Doc type, lang eng... to <head> tag, so be careful not to leave that many blank spaces, you can remove those blank lines to improve your project, it's a necessary change before you get to approve this assignment, for having a code clean and professional.

![image](https://user-images.githubusercontent.com/54015740/120663086-5906a400-c44f-11eb-905e-ca9c67ec45e0.png)